ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake

```diff
@@ -19,9 +19,11 @@
 # Each package name must have been find_package()-ed before.
 # Additionally the exported variables must have a prefix with the same case
 # and the suffixes must be either _INTERFACES or _DEFINITIONS, _INCLUDE_DIRS,
-# _LIBRARIES, and _LINK_FLAGS.
-# If _INTERFACES is not empty it will be used exclusively, otherwise the other
+# _LIBRARIES, _LIBRARY_DIRS, and _LINK_FLAGS.
+# If _INTERFACES is not empty it will be used exclusively, otherwise the otherwil
```

Comment:
Is this change intentional?

Reply:
Not at all. Fixed in 99e6aab2a8c98de405fb735394b0c11929d182a8